uk.co.compucorp.civicrm.hrcomments/CRM/HRComments/Test/Fabricator/Comment.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use CRM_HRComments_BAO_Comment as Comment;
+
+class CRM_HRComments_Test_Fabricator_Comment {
+
+  public static function fabricate($params) {
+    $params = self::mergeDefaultParams($params);
+
+    $comment = Comment::create($params);
```

Comment:
since you're not using `$comment` for anything, what about `return Comment::create($params)`?